src/event-cache/__tests__/EventCache.integ.test.ts

```diff
@@ -25,9 +25,9 @@ describe('EventCache tests', () => {
         const eventCache: EventCache = Utils.createEventCache(config);
 
         // Run
-        eventCache.recordEvent(EVENT1_SCHEMA, {});
+        eventCache.recordEvent(EVENT1_SCHEMA, { field: 'value' });
```

Comment:
nit: Once we revert to allowing empty objects in `recordEvent`, please revert these :-)

Reply:
Done!